### ISSUE TITLE
detach opened program from felix 

### ISSUE THREAD

**Issue by riotearring** [kyoheiu/felix#150]
Hello, I thanks a lot for this really geat utility. 
I've been using it a lot recently and the one disadventage that I find to it is the fact that the opened process depends on felix staying opened 
i.e. : If I open some .pdf file with libreoffice or zathura using felix and then close felix, libreoffice is terminated as well. 
This whether Enter or :o has been used to open the file. 
I believe felix would be even more useful if either : 
- opened process "swallowed" felix instance, therfore hiding it, and not allowing it to be closed but I'm afraid this is a window manger feature 
or 
- opened process was separated from felix process, therefore not requiring felix to run in the background. This is what happens with ranger (https://github.com/ranger/ranger) but not with joshuto (an another rust program)

I might have missed something in the config file and if so sorry for the annoyance, otherwise I am happy to help with some testing but my coding skills are non existent. 

Thanks again for your time !

**Comment by kyoheiu** [comment]
Hm, weird. In my environment, by setting `[exec]` in config.yaml like following...

```
exec:
  zathura:
    [pdf]
```

and by pressing `o` on the item you'd like to open by that command, the new window appears and even if felix/terminal is closed, that window remains open.

Could you tell me your environment and the version of felix (which you can check by `fx --version`)? Mine is below:

```
OS: Arch Linux x86_64
Kernel: 6.0.9-arch1-1
WM: i3
Terminal: alacritty
```

**Comment by riotearring** [comment]
Hello, so here is my system and it appears that both are very similar, only signicative difference probably beaing wayland vs X11

OS: Void Linux x86_64
Kernel: 6.0.9_1
WM: sway
Terminal: alacritty

felix v2.1.0: Up to date.


So i I tried opening zathura and then closing felix on i3 WM, and issus is the same. 
So issue probably comes from config, 
here is the relevant part of mine 

``
default: nvim

exec:
  feh:
    ["jpg", "jpeg", "png", "gif", "svg"]
  zathura: 
    ["pdf"]
libreoffice:
    ["ods", "odt", "odp", "odg", "xlsx", "docx", "dotx", "pptx"]
``

so tried removing this part and only using 

``
exec:
  zathura:
    [pdf]
``

And the issue remaines
No idea of what to do next for nom

**Comment by kyoheiu** [comment]
Thanks for the detail!
The spawned process may wok differently in Wayland. I will look into it.

**Comment by kyoheiu** [comment]
I don't have Wayland environment, but PR #152 may fix this issue. Could you test it?

```
git clone https://github.com/kyoheiu/felix
cd felix
git checkout feature-nix-fork
cargo run 
```

Or if you have `gh`, `gh pr checkout 152` just works.

**Comment by riotearring** [comment]
Just got to testing it, 
Using this method : 
`git clone https://github.com/kyoheiu/felix
cd felix
git checkout feature-nix-fork
cargo run `

it works perfectly using system described above, thanks a lot !

**Comment by kyoheiu** [comment]
merged #152
